gpu4pyscf/lib/cupy_helper/CMakeLists.txt

```diff
@@ -31,6 +31,49 @@ add_library(cupy_helper SHARED
 
 set_target_properties(cupy_helper PROPERTIES
   LIBRARY_OUTPUT_DIRECTORY ${PROJECT_SOURCE_DIR})
-find_package(CUDAToolkit)
+#find_package(CUDAToolkit)
```

Comment:
It is unnecessary to find CUDAToolkit

Reply:
So that's why I commented it out - is it safe to delete?

Reply:
Um wait, I tried on my local machine, CUDAToolkit needs to be found to correctly link the cublas and cusolver. But on HPC it is different from CUDAToolkit so there is the if...else... command below this. I am moving this to the else part, i.e. if NVHPC cmake file is not found.

Reply:
OK. I see.  Keep the one worked for you then.